roles/ngi_reports/templates/ngi_reports_sthlm.conf.j2

```diff
@@ -0,0 +1,32 @@
+[ngi_reports]
+ngi_node = {{ site }}
+support_email = support@ngisweden.se
+[organism_names]
+hg19 = Homo_sapiens
+hg18 = Homo_sapiens
+mm9 = Mus_musculus
+mm8 = Mus_musculus
+Zv8 = Danio_rerio
+Zv9 = Danio_rerio
+Zv10 = Danio_rerio
+rn4 = Rattus_norvegicus
+rn5 = Rattus_norvegicus
+sacCer2 = Saccharomyces_cerevisiae
+canFam2 = Canis_lupus_familiaris
+canFam3 = Canis_lupus_familiaris
+dm3 = Drosophila_melanogaster
+[quality_threshold_hiseq]
+50 : 85
+100 : 80
+125 : 80
+150 : 75
+[quality_threshold_hiseqx]
+<150 : 80
+>150 : 75
+[quality_threshold_miseq]
+<100 : 80
+100 : 75
+150 : 70
+>250 : 60
+[log]
+log_dir : {{ ngi_pipeline_log_sthlm }}
```

Comment:
These vars are defined like 

```
ngi_pipeline_log_sthlm: "{{ ngi_pipeline_sthlm_path }}/log/ngi_pipeline.log"
```

Does that really match how `log_dir` in ngi_reports should be set?